src/storage/src/row_serde/row_serde_util.rs

```diff
@@ -75,3 +75,22 @@ pub fn streaming_deserialize(data_types: &[DataType], mut row: impl Buf) -> Resu
 
     Ok(Row(values))
 }
+
+/// used for streaming table partial deserialize
+pub fn streaming_deserialize(
+    data_types: &[DataType],
+    mut row: impl Buf,
+    value_indices: &[usize],
```

Comment:
As mentioned before, recommend to define a struct for it and wrap the `data_types` and `value_indices` inside of it.

Reply:
BTW, I saw there is a function named `batch_deserialize`: 

https://github.com/risingwavelabs/risingwave/blob/dab77d1428186c39752ac74be2d16d89f11ccdb3/src/storage/src/row_serde/row_serde_util.rs#L65

And the major difference between it and `streaming_deserialize` is whether to taking a `column_mapping`. So after this PR, can these be unified into one serializer?

Reply:
Let's just make `data_types` in state table be the  data_types of `value_columns` instead of the whole columns, so that for `streaming_deserialize`, it's no need to pass a field `value_indices`, and just keep it as before.